api4/ldap_test.go

```diff
@@ -112,3 +198,43 @@ func TestMigrateIdLdap(t *testing.T) {
 		CheckNotImplementedStatus(t, resp)
 	})
 }
+
+func TestUploadPublicCertificate(t *testing.T) {
+	th := Setup(t)
+	defer th.TearDown()
+
+	// sth.th.App.UpdateConfig(func(cfg *model.Config) { *cfg.SamlSettings.IdpCertificateFile = app.SamlIdpCertificateName })
+	// defer sth.th.App.UpdateConfig(func(cfg *model.Config) { *cfg.SamlSettings.IdpCertificateFile = "" })
```

Comment:
Are we able to delete these commented code or do we need it?

Reply:
Deleted